### ISSUE TITLE
Wrong metadata property name when configuring redis configuration component 

### ISSUE THREAD

**Issue by RJvH** [dapr/docs#1944]
**Describe the issue**
When following the docs configuring the redis configuration component an error will be thrown when running dapr : "redis store error: missing host address"

**URL of the docs**
This doc is indicating to use "host" property in the metadata properties
https://docs.dapr.io/developing-applications/building-blocks/configuration/howto-manage-configuration/

The go code seems to check for a "redisHost" metadata property
https://github.com/dapr/components-contrib/blob/a21897bc817630e30a7d668e1d5d9cea21ad706c/configuration/redis/redis.go

**Expected content**
Docs or code is wrong :) One of them should be fixed.

I got it working using this config:
```apiVersion: dapr.io/v1alpha1
kind: Component
metadata:
  name: redis-configuration
spec:
  type: configuration.redis
  metadata:
   - name: redisHost
     value: localhost:6379
```

**Comment by yaron2** [comment]
You are correct - can you please open a PR that fixes this against the v1.5 branch?

**Comment by yaron2** [comment]
NM, done: https://github.com/dapr/docs/pull/1947
